org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsd/XSDValidationExtensionsTest.java

```diff
@@ -222,6 +222,28 @@ public void s4s_elt_invalid_content_3WithSelfClosingTag() throws BadLocationExce
 		testCodeActionsFor(xml, d, ca(d, te(2, 1, 2, 13, "")));
 	}
 
+	@Test
+	public void src_annotation_invalid_tag_() throws BadLocationException {

```

Comment:
Please add  2 new tests

 *  with xs:documentation.
 * with xs:appInfo which have no end tag (<xs:annotation><xs:appInfo></xs:annotation>)

Reply:
Hello, I'm not too sure what you mean. Doesn't this line test for xs:documentation? https://github.com/eclipse/lemminx/blob/5782e0cfe5790c47a30eb3592bc4cd5335d9250a/org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsd/XSDValidationExtensionsTest.java#L240-L242

Reply:
Sorry I though there were too the same error for documentation, please ignore my comment.

Reply:
what about test with with xs:appInfo which have no end tag (<xs:annotation><xs:appInfo></xs:annotation>)?